tests/unit/testServiceConfiguration.js

```diff
@@ -24,546 +24,546 @@ var logger = require('../../src/logger.js')({
   logLevel: isNumber(level) ? level : 4
 });
 
-function makeTest(serviceName, serviceVersion,
-                  moduleName, moduleVersion,
-                  functionName, testFunction) {
-  return function(t) {
-    // store the old environment
-    var oldServiceName = process.env.GAE_SERVICE || '';
-    var oldServiceVersion = process.env.GAE_VERSION || '';
+describe('Testing service configuration', function () {
+  function makeTest(serviceName, serviceVersion,
```

Comment:
I think that's fair, but I would consider this a follow-on update. The tests work and achieve the desired functionality using mocha